internal/commands/waypoint/agent/run.go

```diff
@@ -175,8 +176,13 @@ func runOp(
 		if err != nil {
 			log.Error("unable to register action as starting", "error", err)
 		} else {
+			if resp != nil {
+				if resp.Payload != nil {
+					sequenceNum = resp.Payload.Sequence
+				}
+			}
 			defer func() {
-				log.Info("reporting action run ended", "status", status, "status-code", statusCode)
+				log.Info("reporting action run ended", "status", status, "status-code", statusCode, "sequence", sequenceNum)
```

Comment:
We should be more specific here just so it's clear:

```suggestion
				log.Info("reporting action run ended", "status", status, "status-code", statusCode, "action-run-sequence", sequenceNum)
```